README.md

```diff
@@ -14,11 +14,11 @@ If you're running in a Ruby environment with Bundler and [Sprockets](https://git
 (e.g., [Middleman](https://middlemanapp.com/), [Rails](http://rubyonrails.org/)), add 
 the following line to your `Gemfile` to install from the `master` branch:
 
-    gem "chef-web-core", :git => "https://github.com:chef/chef-web-core"
+    gem 'chef-web-core', git: 'https://github.com/chef/chef-web-core.git'
```

Comment:
Boo! hashrockets and double quotes! Go ahead and and change it but you're a hipster. :)